software/wmla120_ansible/gather_facts.yml

```diff
@@ -0,0 +1,24 @@
+---
```

Comment:
I made some minor changes to get this to run.  I think if you want to have the play named, you may need to put the tasks: key at the same level as the play name.
#- name: Gather localhost facts
- hosts: localhost
  gather_facts: True
  gather_timeout: 10
  tasks: # []
  - name: Gather subset of localhost facts
    setup:
      gather_subset:
        - '!all'
        - min
        - network

#- name: Gather client facts
- hosts: all
  gather_facts: True
  gather_timeout: 10
  tasks: # []
  - name: Gather subset of client facts
    setup:
      gather_subset:
        - '!all'
        - min
        - network